i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md

```diff
@@ -15,6 +15,15 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
 
 ## Centreon MAP
 
+### 22.04.10
+
+Release date: `September 18, 2023`
+
+#### Bug fixes
+
+- [Packaging] Fixed a regression in centreon-map-server and centreon-map-engine packaging that caused a broken dependency situation with Java.
```

Comment:
As there are 2 different chapters between Engine and Legacy, I will only mention the Engine package here.